docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,617 @@
+
+This will be done creating a custom package, following similar steps as in this `ROS 2 tutorial <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Tutorials/Intermediate/Monitoring-For-Parameter-Changes-CPP.html>`_ to be able to change a node's parameter, and respond to that change by changing the Partition QoS of the publisher.
+
+The ROS MiddleWare (see `ROS 2 RMW <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Concepts/About-Different-Middleware-Vendors.html?highlight=RMW#default-rmw-implementation>`_) provides APIs to get handles to the objects of the inner DDS implementation.
+That is needed to be able to change the mutable Qos.
+Thus, this tutorial will also show how to use that powerful tool.
+For another demo on how to access inner RMW entities, see `demo_nodes_cpp_native <https://github.com/ros2/demos/tree/master/demo_nodes_cpp_native>`_.
+
+Prerequisites
+-------------
+
+The first prerequisite is to have Vulcanexus Humble installed (see `Linux binary installation <https://docs.vulcanexus.org/en/latest/rst/installation/linux_binary_installation.html>`_ or `Linux installation from sources <https://docs.vulcanexus.org/en/latest/rst/installation/linux_source_installation.html>`_).
+
+Also before starting this tutorial, you should be familiar with creating a workspace and creating a package, as well as familiar with parameters and their function in a ROS 2 system.
+We recommend to first complete the following tutorials:
+
+    * :ref:`Modifying Ownership and Ownership Strength QoS Policy <tutorials_qos_ownership_ownership>`
+
+    * `Writing a simple publisher and subscriber (C++) <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Tutorials/Beginner-Client-Libraries/Writing-A-Simple-Cpp-Publisher-And-Subscriber.html>`_
+
+    * `Understanding parameters <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Parameters/Understanding-ROS2-Parameters.html>`_
+
+    * `Using parameters in a class (C++) <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Tutorials/Beginner-Client-Libraries/Using-Parameters-In-A-Class-CPP.html>`_
+
+    * `Monitoring for parameter changes (C++) <https://docs.vulcanexus.org/en/latest/ros2_documentation/source/Tutorials/Intermediate/Monitoring-For-Parameter-Changes-CPP.html>`_
+
+Create a package
+----------------
+
+First, open a new terminal and source your ROS 2 installation so that ros2 commands will work.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+
+
+We will create a package in a new workspace folder named `ros2_ws` in `~/`.
+
+.. code-block:: bash
+
+    mkdir ~/ros2_ws && mkdir ~/ros2_ws/src && cd ~/ros2_ws/src
+    ros2 pkg create --build-type ament_cmake vulcanexus_change_mutable_qos --dependencies rclcpp
```

Comment:
Please break the line into a multiline command

Reply:
Done